src/client/models/client/ClientOptions.ts

```diff
@@ -4,10 +4,29 @@ export namespace ClientOptions {
         public useHttps?: boolean;
         public requestHost?: string;
         public timeout?: number;
-        constructor(useHttps?: boolean, requestHost?: string, timeout?: number) {
+        public proxy?: ProxyConfiguration 
+
+        constructor(useHttps?: boolean, requestHost?: string, timeout?: number, proxy?: ProxyConfiguration) {
```

Comment:
We try to do our best to make the library as fexible as possible.

Proxy configuration is something that is not a common and general setting of the Postmark library or underlying http client. Http client can be minimal, especially if it's not the default axios one. Due that, we do not plan to have more than standard options set in the constructor.

http client is available publicly if we want it to do more than the usual, so we should be setting this on the client for now.